app/src/features/observations/TemplateObservationPage.test.tsx

```diff
@@ -0,0 +1,88 @@
+import { cleanup, render, waitFor } from '@testing-library/react';
+import { createMemoryHistory } from 'history';
+import { useBiohubApi } from 'hooks/useBioHubApi';
+import { IGetProjectForViewResponse } from 'interfaces/useProjectApi.interface';
+import { IGetSurveyForViewResponse } from 'interfaces/useSurveyApi.interface';
+import React from 'react';
+import { Route, Router } from 'react-router';
+import { getProjectForViewResponse } from 'test-helpers/project-helpers';
+import { getSurveyForViewResponse } from 'test-helpers/survey-helpers';
+import TemplateObservationPage from './TemplateObservationPage';
+
+const history = createMemoryHistory({ initialEntries: ['/projects/1/surveys/1/observations/create'] });
```

Comment:
why is this the initial history entry? shouldnt it be the `/template` one all along?

Reply:
Mostly because the tests don't care, currently. This whole page is just a temporary thing while we figure out the real workflow, and ideally can be re-used  (or at least parts of it).

Reply:
hmm
ok well if it doesnt care then lets just make it /template so it is less confusing for someone else looking at this test file or if they make a change to the file itself and are trying to update tests in future

Reply:
can do